composer.json

```diff
@@ -14,5 +15,10 @@
         "psr-4": {
             "Neos\\RedirectHandler\\": "Classes"
         }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.0.x-dev"
+        }
```

Comment:
this is to avoid having a 1.0 branch right? if so I don't think it will work since it will mean that the package can only be installed with Flow 3.2+ and not with `dev-master`.. that's why we have branches almost similar for the other packages..